app/experimenter/nimbus-ui/src/index.tsx

```diff
@@ -24,7 +25,7 @@ try {
     <React.StrictMode>
       <AppErrorBoundary>
         <ApolloProvider client={createApolloClient()}>
-          <App basepath="/experiments/nimbus" />
+          <App basepath={BASE_PATH} />
```

Comment:
Oh and now you can ignore my comment from like 5 seconds ago, because this resolves it :)